snakefile.py

```diff
@@ -217,6 +217,7 @@
     params:
         script = os.path.join(SCRIPTS_DIR, 'vcfTocsv.py')
     log: os.path.join(LOG_DIR, 'vcf2csv_{sample}.log')
+<<<<<<< HEAD
```

Comment:
Please rebase your changes and edit the commits that introduce these conflict markers.